packages/kotti-ui/source/kotti-popover/index.ts

```diff
@@ -21,3 +22,13 @@ export const KtPopoverItem = attachMeta(makeInstallable(KtPopoverItemVue), {
 	},
 	typeScript: null,
 })
+
+export const KtPopoverIconTextItem = attachMeta(
+	makeInstallable(KtPopoverIconTextItemVue),
+	{
+		addedVersion: '3.0.0',
```

Comment:
that's not the version you're about to support this on .. 
it's going to be `3.0.0-alpha.19` if nobody publishes before we merge this but double check the last released version and adjust to it

Reply:
Please use `3.0.0` though